File: ru.kmorozov.gbd.core/src/main/java/ru/kmorozov/gbd/core/logic/model/book/google/AdditionalInfo.java
package ru.kmorozov.gbd.core.logic.model.book.google;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by km on 28.11.2015.
 */
class AdditionalInfo implements Serializable {

    @SerializedName("[JsonBookInfo]")
    private JsonBookInfo jsonBookInfo;
}

File: ru.kmorozov.gbd.db/src/main/java/ru/kmorozov/db/core/logic/model/book/google/AdditionalInfo.kt
package ru.kmorozov.db.core.logic.model.book.google

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

import java.io.Serializable

/**
 * Created by km on 28.11.2015.
 */
class AdditionalInfo : Serializable {

    @Expose
    @SerializedName("[JsonBookInfo]")
    val jsonBookInfo: JsonBookInfo? = null
}
